src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/partially_quiet.phpt

```diff
@@ -0,0 +1,32 @@
+--TEST--
+Test DeprecationErrorHandler quiet on everything bug self deprecations with no
```

Comment:
typo

Reply:
it's not a bug, it's a typo :sweat_smile: